tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
`autogen.generator` needs to read `cypari2/paridecl.pxd` and `cypari2/declinl.pxi` in order to run doctests, so it fails if we do not chdir.